src/glossary.rst

```diff
@@ -707,15 +713,58 @@ Glossary
       * :term:`run directory`
 
    source directory
-      Any directory where workflows are written and stored in preparation for
-      installation with ``cylc install`` or reinstallation with
-      ``cylc reinstall``.
+      Any directory where :term:`workflows <workflow>` are written and stored
+      in preparation for installation with ``cylc install`` or reinstallation
+      with ``cylc reinstall``.
```

Comment:
Might be worth mentioning the search path can be configured in `global.cylc[install]source dirs`